<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TotalTable</class>
 <widget class="QDialog" name="TotalTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="totalTable_lineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>132</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>十六进制值</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget2">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>120</y>
     <width>212</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>动作表长度：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="actionTable_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget3">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>150</y>
     <width>212</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>动态表长度：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="dynamicTable_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget4">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>180</y>
     <width>212</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>监控表长度：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="monitoringTable_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>216</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>总表长度：</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget5">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>90</y>
     <width>212</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>静态表长度：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="staticTable_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>221</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_10">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_11">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget7">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>82</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="totalStatic_pushButton">
      <property name="text">
       <string>获取静态表</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="totalAction_pushButton">
      <property name="text">
       <string>获取总动作表</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="totalDynamic_pushButton">
      <property name="text">
       <string>获取总动态表</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="totalMonitoring_pushButton">
      <property name="text">
       <string>获取总监控表</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="total_pushButton">
      <property name="text">
       <string>获取总表</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget8">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>267</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <property name="spacing">
     <number>20</number>
    </property>
    <item>
     <widget class="QPushButton" name="tableHead_pushButton">
      <property name="text">
       <string>生成表头</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="totalTable_pushButton">
      <property name="text">
       <string>生成总表</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="head_total_pushButton">
      <property name="text">
       <string>表头+3总表</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>长度</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
